### ISSUE TITLE
Re-exporting interface in namespace via export import is considered a "variable"

### ISSUE THREAD

**Issue by MathBunny** [TypeStrong/typedoc#1353]
## Search terms

namespace, aliasing, import, export, re-exporting, interface

## Expected Behavior

TypeDoc will produce a web-page consisting of two interfaces listed for the following code:
```ts
/**
 * Sample namespace exported that leverages aliasing. 
 */
export namespace admin.subnamespace{
  export import DemoInterface = demoInterfaceApi.DemoInterface; // DOES NOT WORK!
  export interface DemoClass extends demoClassApi.DemoClass {}
}
```

## Actual Behavior

The following appears with variables as one of the sections:
![](https://camo.githubusercontent.com/22e5ab33c6e2dc42726cae8d26e434f842ebd8ac/68747470733a2f2f692e696d6775722e636f6d2f44544b4f58367a2e706e67)

## Steps to reproduce the bug

Visit my repository: [github.com/MathBunny/typedoc-namespace-bug-demo](http://github.com/MathBunny/typedoc-namespace-bug-demo)

- All the source code is found in `src`
- You can test and build via `npm run build && npm run docs`

This seems related to #1157 but I'm not sure if `export import` is fully supported yet or what the plans are for it; the issues aren't very clear on what is and what is not supported yet.

## Environment
- Typedoc version: `0.18.0`
- TypeScript version: `3.9.5`
- Node.js version: `v12.18.1`
- OS: macOS High Sierra `10.13.3`

**Comment by MathBunny** [comment]
Adding to this, it also appears that enums aren't recognized correctly either, and they're labelled as variables. 

```ts
export const AppPlatform: typeof appMetadataApi.AppPlatform = appMetadataApi.AppPlatform;
```

Here, `appPlatform` is an enum. The other option is to do:

```ts
export import AppPlatform = appMetadataApi.AppPlatform;
```

This doesn't seem to work either.

**Comment by Gerrit0** [comment]
```ts
export import DemoInterface = demoInterfaceApi.DemoInterface;
```

TIL this is legal...  Every other time I've only seen `import =` used to import something as a variable, as in the handbook - https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require. Looks like a bug.

```ts
export const AppPlatform: typeof appMetadataApi.AppPlatform = appMetadataApi.AppPlatform;
```

This is a variable. TypeDoc's output is correct here. The `export import` variant should be parsed as a re-export, but isn't.

The chanes in #1157 were focused on the more standard `export { a, b }` used in modules, not namespaces.

Right now, TypeDoc has `ImportEquals` handled in the [variable converter](https://github.com/TypeStrong/typedoc/blob/master/src/lib/converter/nodes/variable.ts#L29), it should really get its own converter which converts as re-exports.

**Comment by MathBunny** [comment]
That makes sense, thanks for following up. Okay, the focus of this issue should remain on the `export import` aspect.

**Comment by Gerrit0** [comment]
This is fixed in the 0.20 beta running typedoc on the following:

```ts
export namespace NamespaceA {
    export interface InterfaceA { }
    //export type renamedInterfaceA =  InterfaceA; // WA to typedoc v0.15
}
export namespace NamespaceB {
    export interface InterfaceA { }
}
export namespace NamespaceC {
    export import renamedInterfaceA = NamespaceA.InterfaceA;
    export import InterfaceA = NamespaceB.InterfaceA;

    export class TestClass {
        testMethod1(options: renamedInterfaceA): any {}
        testMethod2(options: InterfaceA): any {};
    }
}
``` 

and then viewing NamespaceC shows the following, with the InterfaceA links going to the appropriate page:

![image](https://user-images.githubusercontent.com/19329837/100490267-6ffebf00-30d7-11eb-9ae5-816fd3122b75.png)
